tools/ci/check_grep.sh

```diff
@@ -45,6 +45,11 @@ if grep -P --exclude='__byond_version_compat.dm' '(\.proc\/)|(CALLBACK\(.*proc\/
     echo "ERROR: Outdated proc reference use detected in code, please use proc reference helpers."
     st=1
 fi;
+# Check for to_chat procs lacking proper arguments
+if grep -P '\to_chat("/' code/**/*.dm; then
```

Comment:
This should probably be done in check_grep2.py

Reply:
Done